README.md

```diff
@@ -20,20 +20,48 @@ npm install @kava-labs/javascript-sdk
 
 ### Mainnet
 
-- Chain ID: kava-3
-- REST API endpoint: https://kava3.data.kava.io
+- Chain ID: kava-7
+- REST API endpoint:  https://api.kava.io
 - BEP3 deputy's Kava address: kava1r4v2zdhdalfj2ydazallqvrus9fkphmglhn6u6
 - BEP3 deputy's Binance Chain address: bnb1jh7uv2rm6339yue8k4mj9406k3509kr4wt5nxn
```

Comment:
Likewise, can remove this line now that deputy addresses are below

Reply:
fixed in [7a7494d826c34ffc25fce18df8c2e9de88059dbe](https://github.com/Kava-Labs/javascript-sdk/commit/7a7494d826c34ffc25fce18df8c2e9de88059dbe)